libs/semanticsearch/include/semanticsearch/schema/vocabulary_instance.hpp

```diff
@@ -19,7 +19,7 @@
 
 #include <functional>
 #include <map>
-#include <string>
+#include <memory> #include <string>
```

Comment:
newline

Reply:
Weird note sure where that came from, but fixed